packages/voice/src/util/Secretbox.ts

```diff
@@ -5,6 +5,31 @@ interface Methods {
 }
 
 const libs = {
+	'sodium-native': (sodium: any): Methods => ({
+		open: (buffer: Buffer, nonce: Buffer, secretKey: Uint8Array) => {
+			if (Boolean(buffer)) {
+				// eslint-disable-next-line @typescript-eslint/no-unsafe-member-access
+				const output = Buffer.allocUnsafe(buffer.length - sodium.crypto_box_MACBYTES);
+				// eslint-disable-next-line @typescript-eslint/no-unsafe-member-access, @typescript-eslint/no-unsafe-call
+				if (sodium.crypto_secretbox_open_easy(output, buffer, nonce, secretKey)) return output;
+			}
+			return null;
+		},
+		close: (opusPacket: Buffer, nonce: Buffer, secretKey: Uint8Array) => {
+			// eslint-disable-next-line @typescript-eslint/no-unsafe-member-access, @typescript-eslint/no-unsafe-argument, @typescript-eslint/restrict-plus-operands
+			const output = Buffer.allocUnsafe(opusPacket.length + sodium.crypto_box_MACBYTES);
+			// eslint-disable-next-line @typescript-eslint/no-unsafe-member-access, @typescript-eslint/no-unsafe-call
+			sodium.crypto_secretbox_easy(output, opusPacket, nonce, secretKey);
+			return output;
+		},
+		random: (n: any, buffer?: Buffer) => {
+			// eslint-disable-next-line @typescript-eslint/no-unsafe-argument
+			if (!buffer) buffer = Buffer.allocUnsafe(n);
```

Comment:
```suggestion
		// eslint-disable-next-line @typescript-eslint/no-unsafe-argument
		random: (n: any, buffer: Buffer = Buffer.allocUnsafe(n)) => {
```
I guess you could just do this instead, since if param already has an initializer, it's optional.

Reply:
I just copied it from the sodium implementation down below lol